packages/forms-web-app/src/pages/process-guide/router.js

```diff
@@ -19,10 +19,18 @@ const { decisionURL } = require('./decision/config');
 const { postDecisionURL } = require('./post-decision/config');
 
 const { addSteps } = require('./_middleware/add-steps');
+const {
+	addProcessGuideTranslationsMiddleware
+} = require('./_middleware/add-process-guide-translations-middleware');
 
 const processGuideRouter = express.Router();
 
-processGuideRouter.get(processGuideURL, addSteps, getProcessGuideController);
+processGuideRouter.get(
+	processGuideURL,
+	addSteps,
+	addProcessGuideTranslationsMiddleware,
```

Comment:
Are we going to have page specific translations or import all the translations for each route? If the latter is the choice we could add the translations once to the process guide router before the routes.